docs/scenario_guides/scenario_secret_passing.rst

```diff
@@ -0,0 +1,32 @@
+.. _secret_passing:
+
+Passing Secrets
+===============
+
+Sometimes it may be useful to use `build secrets <https://docs.docker.com/build/building/secrets/>`_ when
+creating an Execution Environment. This can be done with a combination of the use of :ref:`additional_build_steps`
+within the EE definition file, and the :ref:`extra-build-cli-args` CLI option.
+
+Use the :ref:`extra-build-cli-args` CLI option to pass a build CLI argument that defines the secret:
```

Comment:
@Shrews do you think we should a note here saying that - the secrets are passed in plain text and may be recorded in console logs so be careful using this parameter or something along these lines?

Reply:
Secrets are not being passed in plain text.